site/public/js/notification-sound.js

```diff
@@ -0,0 +1,19 @@
+        }
+        else {
+            this.sound = null;
+        }
+    }
+
+    play() {
+        this.sound ? this.sound.play() : alert(`A queue change occured, but your browser doesn't support playing audio on this page.`);
```

Comment:
Looking at this, if a user activates sound on a computer that does not support and then wanders away for a while and a bunch of queue stuff happens, when they come back, are they then going to have to click through a bunch of alert windows all saying this error message?

Reply:
Good point, I will rework this.

Reply:
My view would be to do this check on attempting to turn on the icon, and perhaps disable / gray out the icon altogether on page load if they cannot play sound with a mouse over text of "Your browser does not support step sound" or something?

Reply:
I reworked this so that the button is now only displayed if we are reasonably sure the browser supports playing the audio.